src/raftstore/store/keys.rs

```diff
@@ -80,6 +80,10 @@ pub fn region_raft_prefix(region_id: u64) -> Vec<u8> {
     key
 }
 
+pub fn region_raft_prefix_len() -> usize {
+    REGION_RAFT_PREFIX_KEY.len() + mem::size_of::<u64>()
```

Comment:
do we need to handle apply raft key, raft state key and raft log key separately?  They have same raft prefix but with different postfix:

```
pub const RAFT_LOG_SUFFIX: u8 = 0x01;
pub const RAFT_STATE_SUFFIX: u8 = 0x02;
pub const APPLY_STATE_SUFFIX: u8 = 0x03;
```